src/components/Pagination.svelte

```diff
@@ -84,14 +93,48 @@
 
 <div class="flex flex-col items-center my-12">
   <div class="flex text-gray-700">
-    <div class="flex h-12 font-medium bg-gray-200">
-      {#each range(lastPage, 1) as page}
+    <div
+      on:click|preventDefault={() => changePage(currentPage !== 1 ? currentPage - 1 : 1)}
+      class="h-12 w-12 mr-1 flex justify-center items-center rounded-full bg-gray-200 cursor-pointer {currentPage === 1 ? 'bg-teal-600 text-white' : ''}">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="100%"
+        height="100%"
+        fill="none"
+        viewBox="0 0 24 24"
+        stroke="currentColor"
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        class="feather feather-chevron-left w-6 h-6">
+        <polyline points="15 18 9 12 15 6" />
+      </svg>
```

Comment:
Could you put this in a seperate svelte file under an `./icons/` directory and import it? This will make it easier to reuse later (and also make this page easier to understand).